inference/hf-text-generation-inference/hf_tgi_inference.py

```diff
@@ -0,0 +1,77 @@
+import json
+from text_generation import Client
```

Comment:
This requires users to install text_generation which is currently not part of the requirements.txt?
Could you rewrite the example to use a more "curl like" approach using the requests package instead of text_generation?

Reply:
Good idea. I'll fix that.